NAMESPACE

```diff
@@ -2,13 +2,12 @@
 
 S3method(autoplot,imputed_trait)
 S3method(fortify,imputed_trait)
-export(fit_trait_distributions)
+export(trait_fit_distributions)
 export(trait_impute)
 export(trait_missing)
 export(trait_np_bootstrap)
 export(trait_parametric_bootstrap)
 export(trait_summarise_boot_moments)
-importFrom(data.table,":=")
```

Comment:
did you take the dependency on data.table out of the DESCRIPTION?

Reply:
Totally forgot.  I've moved it from depends to suggests (where it was originally)

Reply:
I think you added data.table to suggests - it is not in the version on currently github - not a package I use regularly. Please also add you name etc to the author field of the description. Should also add ORCID. 
```
    person(given = "First",
           family = "Last",
           role = c("aut", "cre"),
           email = "first.last@example.com",
           comment = c(ORCID = "YOUR-ORCID-ID"))
```